### ISSUE TITLE
Support offline compression of form media

### ISSUE THREAD

**Issue by grote** [django-compressor/django-compressor#880]
Using [form assets via the Media class](https://docs.djangoproject.com/en/1.11/topics/forms/media/) is a thing.

I am surprised that this does not seem to be supported for offline compression. There's even [a question on stackoverflow](
https://stackoverflow.com/questions/32433609/how-to-use-django-compressor-to-compress-form-media-offline) for it already, but I didn't find anything here in the issue tracker.

The tricky part is probably finding all the form assets that could be included in the templates during offline compression. Maybe [`django.conf.settings.COMPRESS_OFFLINE_CONTEXT`](http://django-compressor.readthedocs.io/en/latest/settings/?highlight=media#django.conf.settings.COMPRESS_OFFLINE_CONTEXT) could help with that.

**Comment by karyon** [comment]
yeah, looks like this is not supported. i don't think any of us maintainers will implement this, but we would welcome pull requests (although i have no idea how complex this would be).
